examples/3d/parameters_amr_blob.toml

```
@@ -0,0 +1,52 @@
+####################################################################################################
+# Simulation
+ndims = 3
+equations = "CompressibleEulerEquations"
+gamma = 1.66667
+initial_conditions = "initial_conditions_blob"
+t_start = 0.0
+t_end   = 2.5
+#restart = true
+#restart_filename = "out/restart_001000.h5"
+
+####################################################################################################
+# Solver
+solver = "dg"
+polydeg = 3
+cfl = 0.9
+n_steps_max = 15000
+volume_integral_type = "split_form"
+surface_flux = "flux_hllc"
+volume_flux = "flux_ranocha"
+pp_limiter_apply = true
+pp_limiter_threshold = 0.0001
+
+####################################################################################################
+# Mesh
+n_cells_max = 500000
+coordinates_min = [-20.0, -20.0, -20.0]
+coordinates_max =  [20.0, 20.0, 20.0]
+initial_refinement_level = 2
+ refinement_patches = [
+   {type = "box", coordinates_min = [-20.0, -10.0, -10.0], coordinates_max = [-10.0, 10.0, 10.0]},
+   {type = "box", coordinates_min = [-20.0, -5.0, -5.0], coordinates_max = [-10.0, 5.0, 5.0]},
+   {type = "box", coordinates_min = [-17.0, -2.0, -2.0], coordinates_max = [-13.0, 2.0, 2.0]},
+   {type = "box", coordinates_min = [-17.0, -2.0, -2.0], coordinates_max = [-13.0, 2.0, 2.0]},
+#   {type = "box", coordinates_min = [-17.0, -2.0, -2.0], coordinates_max = [-13.0, 2.0, 2.0]},
+ ]
+####################################################################################################
+# AMR
+adapt_initial_condition = false
+amr_indicator = "blob"
+amr_interval = 3
+amr_alpha_max = 1.0 
+amr_alpha_min = 0.0001
+
+####################################################################################################
+# I/O
+# save_initial_solution = false
+analysis_interval = 200
+solution_interval = 200
+solution_variables = "primitive"
+restart_interval = 200
+output_format = "hdf5"
```

```suggestion
analysis_interval = 200
solution_interval = 200
restart_interval = 200
```
Remove comments and default values for clarity